output/file.go

```diff
@@ -60,41 +58,47 @@ func makeNonOverlappingFilename(path string) string {
 }
 
 func (f *FileWriter) Download(resp *http.Response) error {
-	// Create new progress bar
-	pb := mpb.New(mpb.WithWidth(60))
-
 	// Create file
 	file, err := os.Create(f.fullPath)
 	if err != nil {
 		return err
 	}
 	defer file.Close()
 
-	// Parameters of th new progress bar
```

Comment:
What is this file about?

And why things are removed and something different added?

Reply:
It is used for sending file through a request, it had some incompatible wasm dependencies.

Fixed the file now added `!wasm` tag to exclude wasm builds for now.